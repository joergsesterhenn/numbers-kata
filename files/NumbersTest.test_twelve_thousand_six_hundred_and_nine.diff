--- NumbersTest.test_twelve_thousand_six_hundred_and_nine.approved.txt
+++ NumbersTest.test_twelve_thousand_six_hundred_and_nine.received.txt
@@ -1 +1 @@
-twelve thousand, six hundred and nine
+twelve thousand, nine hundred and six
