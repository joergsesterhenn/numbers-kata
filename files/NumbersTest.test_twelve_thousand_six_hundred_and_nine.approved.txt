twelve thousand, six hundred and nine
